specifyweb/frontend/js_src/lib/localization/forms.ts

```diff
@@ -1166,4 +1166,7 @@ export const formsText = createDictionary({
     'uk-ua': '(діапазон: {from:string} - {to:string})',
     'de-ch': '(Bereich: {from:string} - {to:string})',
   },
+  record: {
+    'en-us': 'Record',
+  },
```

Comment:
(optional) we already have userText.resource(). Does that work for you?

Reply:
that works!